### DISCUSSION TITLE
What do you know about omnichannel retailing?

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#300] [General]
Please let me know what do you know about it.

**Comment by hernessteve** [comment]
That's funny, cause i heard about this term only few days ago for the first time. I found a great blog about [omnichannel retailing](https://forbytes.com/blog/omnichannel-retailing-tips-for-stores-getting-started-online/) and other useful business topics and spent hours learning new things i didn't have even idea about.  The google describes it as 
" a retailer's efforts to provide a consistent, coordinated customer experience across all possible customer channels, using consistent, universal data" but you might find much more facts about it in the blog.
